bee-node/src/plugins/version_checker.rs

```diff
@@ -31,7 +40,11 @@ impl<N: Node> Worker<N> for VersionChecker {
             );
 
             while ticker.next().await.is_some() {
-                // TODO
+                match is_latest(&client).await {
+                    Ok(false) => warn!("A new version has been released. Please update the node at https://github.com/iotaledger/bee/releases"),
```

Comment:
I'ld change the message to something like:
"A new version has been released. All releases can be found here: https://github.com/iotaledger/bee/releases.".